src/braket/circuits/circuit.py

```diff
@@ -1196,6 +1201,9 @@ def _to_openqasm(
             ]
         )
 
+        if self.global_phase:
+            ir_instructions.append(f"gphase({self.global_phase});")
```

Comment:
Are we ok with combining all the gphase instructions? What if someone has the instruction in multiple places and they expect to see them there? I understand they are equivalent, but we don't remove `x x` from a Circuit in the BDK

Reply:
Yes, that makes sense. I'd love for more people to weigh in here before we decide which approach to use. Thanks for explaining the options